aws/cloudformation/components/database.yml.erb

```diff
@@ -185,6 +186,51 @@
         # exceed 5% of the total database connection capacity of a reader instance.
         MaxConnectionsPercent: 5
 
+  WriterSQLUser: <%= lambda_fn 'SQLUser', {
```

Comment:
This renders a YAML CloudFormation resource whose value is JSON. The `DependsOn` setting does not seem to be working because when deleting a Stack, CloudFormation often tries to delete this Resource *after* the database has been deleted.

``` YAML
WriterSQLUser: {"Type":"Custom::SQLUser","Properties":{"ServiceToken":{"Fn::Join":[":",["arn:aws:lambda",{"Ref":"AWS::Region"},{"Ref":"AWS::AccountId"},"function","SQLUser"]]},"DBServerHost":{"Fn::GetAtt":["AuroraCluster","Endpoint.Address"]},"DBCredentialAdminSecret":{"Ref":"DatabaseSecretAdmin"},"DBCredentialSecret":{"Ref":"DatabaseSecretWriter"},"Privileges":["SELECT","INSERT","UPDATE","DELETE","CREATE","DROP","REFERENCES","INDEX","ALTER","CREATE TEMPORARY TABLES","LOCK TABLES","EXECUTE","CREATE VIEW","SHOW VIEW","CREATE ROUTINE","ALTER ROUTINE","EVENT","TRIGGER"]},"DependsOn":["Aurora0","Aurora1"]}
```